src/server.h

```diff
@@ -1734,7 +1734,7 @@ struct redisServer {
     durationStats duration_stats[EL_DURATION_TYPE_NUM];
 
     /* Configuration */
-    int verbosity;                  /* Loglevel in redis.conf */
+    int verbosity;                  /* Loglevel in placeholderkv.conf */
```

Comment:
This comment just makes no sense, it's probably a holdover form the earliest days of the project.
```suggestion
    int verbosity;                  /* Loglevel verbosity */
```